apps/web/__tests__/test/smoke/wishlistPage.cy.ts

```diff
@@ -0,0 +1,63 @@
+import { paths } from '../../../utils/paths';
```

Comment:
We want to create a new folder structure where we have real smoke tests that just check if the app runs. So that we get early feedback and dont have to wait until all tests finished.

New structure should be /test/feature/wishlistPage.cy.ts 
all the other tests can be left in smoke for now.

Reply:
done